app/screens/welcome/final_screen.tsx

```diff
@@ -1,15 +1,16 @@
 import { ThemedText } from "@/components/theme/ThemedText";
 import { ThemedTextButton } from "@/components/theme/ThemedTextButton";
 import { ThemedView } from "@/components/theme/ThemedView";
-import { useRouter } from "expo-router";
+//import { useRouter } from "expo-router";
 import React from "react";
 import { StyleSheet, Dimensions } from "react-native";
+import App from "@/app/App";
 
 // Get the screen dimensions
 const { width, height } = Dimensions.get("window");
 
 export default function WelcomeConceptScreen() {
-  const router = useRouter();
+  const router = App();
```

Comment:
This isn't centralised, we should have our stack navigation in the index.tsx file